src/main/java/nl/mpcjanssen/simpletask/Simpletask.kt

```diff
@@ -62,6 +62,9 @@ import android.R.id as androidId
 
 class Simpletask : ThemedNoActionBarActivity() {
 
+    private const val EDIT: Boolean = true
+    private const val ADD: Boolean = false
+
```

Comment:
I didn't *need* to do this, but `startAddTaskActivity(EDIT)` makes a lot more sense than `startAddTaskActivity(true)`